bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -8,20 +9,63 @@ import DropdownListTitle from '/imports/ui/components/dropdown/list/title/compon
 import DropdownListSeparator from '/imports/ui/components/dropdown/list/separator/component';
 import DropdownListItem from '/imports/ui/components/dropdown/list/item/component';
 import Icon from '/imports/ui/components/icon/component';
+import Button from '/imports/ui/components/button/component';
+import VideoListItemStats from './video-list-item-stats/component';
 import { styles } from '../styles';
 
+const intlMessages = defineMessages({
+  connectionStatsLabel: {
+    id: 'app.video.stats.title',
+  },
+});
+
 class VideoListItem extends Component {
   constructor(props) {
     super(props);
     this.videoTag = null;
+
+    this.state = {
+      showStats: false,
+    }
+
+    this.toggleStats = this.toggleStats.bind(this);
   }
 
   componentDidMount() {
     this.props.onMount(this.videoTag);
   }
 
+  toggleStats() {
+    this.setState({showStats: !this.state.showStats});
+  }
+
+  getAvailableActions() {
+    const {
+      intl,
+      actions,
+      user,
+    } = this.props;
+
+    return _.compact([
+      <DropdownListTitle className={styles.hiddenDesktop} key="name">{user.name}</DropdownListTitle>,
+      <DropdownListSeparator className={styles.hiddenDesktop} key="sep" />,
+      ...actions.map(action => (<DropdownListItem key={user.id} {...action} />)),
+      (Meteor.settings.public.kurento.enableVideoStats ?
```

Comment:
We try to not use `Meteor.settings` outside of containers, at least make it a prop and set the setting as `defaultValue`.